data/src/main/java/com/my/kizzy/data/rpc/KizzyRPC.kt

```diff
@@ -102,6 +104,22 @@ class KizzyRPC(
         return this
     }
 
+    /**
+     * Party of Rpc
+     *
+     * @param party
+     * @return
+     */
+    fun setPartySize(current: Int?, max: Int?): KizzyRPC {
+        if (current != null && max != null) {
+            this.party = Party(
+                id = "kizzy",
```

Comment:
Could this be a randomly generated string instead of kizzy or maybe have the user provide one. Although dont think adding another field for user to fill in is good. What do you think ?

Reply:
The ID of the party doesn't really matter and aren't displayed on the RPC. They're only used for callbacks when people click on game invitations on Discord.
It could be a random string if you want, but the user would more likely be left confused if this were made into a field.